app/controllers/concerns/body_class_concern.rb

```diff
@@ -24,6 +24,10 @@ def landing_page?
     namespace_name.nil? && controller_name == "pages" && action_name == "index"
   end
 
+  def theme
+    "theme-unset"
```

Comment:
So this can be `theme-light` or `theme-dark`. If it's `theme-unset` then we should go with the browser default.

For now you can just manually change the string here to experiment.